--- loaddap-3.7.2/loaddap.c.orig	2012-05-31 19:22:55.000000000 +0200
+++ loaddap-3.7.2/loaddap.c	2014-03-15 12:32:55.367212832 +0100
@@ -64,6 +64,7 @@
 #define ARCH_GLNX86
 #endif
 
+#include <stdio.h>
 #include <string.h>
 #include <errno.h>
 #ifdef WIN32
--- loaddap-3.7.3/process_values.c.orig	2023-01-19 18:00:14.647338290 +0100
+++ loaddap-3.7.3/process_values.c	2023-01-19 18:00:55.880448244 +0100
@@ -42,6 +42,7 @@
 #include "config.h"
 
 #include <errno.h>
+#include <stdio.h>
 #include <string.h>
 #include <math.h>
 
