--- loaddap-3.7.2/loaddap.c.orig	2012-05-31 19:22:55.000000000 +0200
+++ loaddap-3.7.2/loaddap.c	2014-03-15 12:32:55.367212832 +0100
@@ -64,6 +64,7 @@
 #define ARCH_GLNX86
 #endif
 
+#include <stdio.h>
 #include <string.h>
 #include <errno.h>
 #ifdef WIN32
--- loaddap-3.7.2/extend.c.orig	2012-05-31 19:22:55.000000000 +0200
+++ loaddap-3.7.2/extend.c	2014-03-15 12:36:26.140537321 +0100
@@ -46,6 +46,7 @@
 
 static char id[]= { "$Id: extend.c 21443 2009-09-02 08:52:42Z dan $" };
 
+/* cannot use <stdbool.h>, it conflicts with mex.h for C */
 #include <stdio.h>
 #include <string.h>
 
@@ -840,7 +841,7 @@
 	int status, nFields, idx, i, j;
 	int numUniqueVariables;
 	int start, next, count;
-	bool Found = false;
+	bool Found = FALSE;
 	const char *name;
 	MLVars *structArray;
 	mxArray *ret_array;
@@ -875,7 +876,7 @@
 #endif
 		for (i=0; i<count; i++) {
 			if (strcmp(names[i], name)==0) {
-				Found = true;
+				Found = TRUE;
 			}
 		}
 		if ( !Found ) {
--- loaddap-3.7.2/process_values.c.orig	2012-05-31 19:22:54.000000000 +0200
+++ loaddap-3.7.2/process_values.c	2014-03-15 12:36:47.487203091 +0100
@@ -44,6 +44,7 @@
 static char id[] not_used = {"$Id: process_values.c 21569 2009-09-30 04:12:29Z jimg $"};
 
 #include <errno.h>
+#include <stdio.h>
 #include <string.h>
 
 #include <mex.h>
